libcudacxx/test/support/test_iterators.h

```diff
@@ -700,6 +700,7 @@ class contiguous_iterator
   void operator,(T const&) = delete;
 };
 static_assert(cuda::std::random_access_iterator<contiguous_iterator<int*>>, "");
+static_assert(cuda::std::contiguous_iterator<contiguous_iterator<int*>>, "");
```

Comment:
This is a fun one, MSVC was bailing in C++17 mode to determine `contiguous_iterator` of `move_iterator<contiguous_iterator>` because it could not find resolve the `::cuda::std::contiguous_iterator<contiguous_iterator>`